config/runtime.exs

```diff
@@ -7,8 +7,11 @@ import Config
 
 alias FzCommon.{CLI, FzInteger, FzString, FzKernelVersion, FzNet}
 
-# external_url is important
-external_url = System.get_env("EXTERNAL_URL", "https://localhost") |> FzNet.to_complete_url
+# external_url is important, so fail fast here if we can't parse
+{:ok, external_url} =
+  System.get_env("EXTERNAL_URL", "https://localhost")
```

Comment:
as we discussed shouldn't we use `System.fetch_env!` here so to make sure we fail and we don't default to localhost?

Reply:
Yeah, hm let me update this. I was going back and forth on that.